### PR TITLE
Features/shp 67 change onlineshopapi namespaces

### PR DESCRIPTION
**Pull request by Yugmund**

### CODE REVIEW

File: SHP.OnlineShopAPI.Tests/ControllerTests/CategoryControllerTests.cs [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
+using SHP.OnlineShopAPI.Controllers;
 using OnlineShopAPI.DTO.Category;
 using OnlineShopAPI.Mapping;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 using Xunit;
 
-namespace OnlineShopAPI.Tests
+namespace SHP.OnlineShopAPI.Tests
```
Comment (4AK1LLA) [inline SHP.OnlineShopAPI.Tests/ControllerTests/CategoryControllerTests.cs]: Some changes needed in Tests namespaces, for example, SHP.OnlineShopAPI.Tests -> SHP.OnlineShopAPI.Tests.ControllerTests

File: SHP.OnlineShopAPI.Tests/ServiceTests/PhotoServiceTests.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@
 using System.Threading.Tasks;
 using Xunit;
 
-namespace OnlineShopAPI.Tests
+namespace SHP.OnlineShopAPI.Tests
```
Comment (4AK1LLA) [inline SHP.OnlineShopAPI.Tests/ServiceTests/PhotoServiceTests.cs]: SHP.OnlineShopAPI.Tests -> SHP.OnlineShopAPI.Tests.ServiceTests

File: SHP.OnlineShopAPI.Tests/ControllerTests/CategoryControllerTests.cs (hunk 1 above)
Comment (Yugmund) [reply SHP.OnlineShopAPI.Tests/ControllerTests/CategoryControllerTests.cs]: Fixed

File: SHP.OnlineShopAPI.Tests/ServiceTests/PhotoServiceTests.cs (hunk 2 above)
Comment (Yugmund) [reply SHP.OnlineShopAPI.Tests/ServiceTests/PhotoServiceTests.cs]: Fixed

**Review by 4AK1LLA — APPROVED** [review]

**Pull request merged**